src/main/scripts/org/reaktivity/specification/http2/rfc7540/connection.management/connection.established/server.rpt

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newSourceInputRef 1234L # external
```

Comment:
These types of external properties should really be initialized to a random value.

Do you think it would make sense to add such functions to the `nukleus` transport implementation so they are automatically available to script authors?